src/content/helpers/faceit-api.js

```diff
@@ -63,9 +63,9 @@ export const getUser = userId => fetchApiMemoized(`/core/v1/users/${userId}`)
 export const getPlayer = nickname =>
   fetchApiMemoized(`/core/v1/nicknames/${nickname}`)
 
-export const getPlayerMatches = (userId, game, size = 21, page = 0) =>
+export const getPlayerMatches = (userId, game, size = 20) =>
   fetchApiMemoized(
-    `/stats/v1/stats/time/users/${userId}/games/${game}?page=${page}&size=${size}`
+    `/stats/api/v1/stats/time/users/${userId}/games/${game}?size=${size}`
```

Comment:
Where did you get this endpoint from? Afaik this is not how FACEIT are naming their endpoints.

Reply:
I changed this to fix memoization. 

Line 86 uses this API too:
>  let averageStats = await fetchApiMemoized(
    `/stats/api/v1/stats/time/users/${userId}/games/${game}?size=${size}`
  )

I shall change them both to make sure that there are no multiple requests done.